movie_search/models.py

```diff
@@ -1,10 +1,9 @@
-from django.db import models
-
 # Create your models here.
 from django.db import models
 
+
 class Genre(models.Model):
-    name = models.CharField(max_length=200)
+    name = models.CharField(max_length=200, null=True, blank=True, unique=True)
```

Comment:
🎉 - this should solve the issue we worked on yesterday (you would have to wipe out genres table and run the fixture again, if you run the fixture again do you get a crash?)